tests/integration/test_github/test_configuration/test_invalid.py

```diff
@@ -0,0 +1,138 @@
+from wazuh_testing.utils.configuration import load_configuration_template
+from . import CONFIGS_PATH, TEST_CASES_PATH
+
+# Marks
+pytestmark = pytest.mark.tier(level=0)
+
+# variables
+force_restart_after_restoring = True
+
+# Configuration and cases data.
+configs_path = Path(CONFIGS_PATH, 'config_invalid_configuration.yaml')
+cases_path = Path(TEST_CASES_PATH, 'cases_invalid_configuration.yaml')
+
+# Test configurations.
+config_parameters, test_metadata, test_cases_ids = get_test_cases_data(cases_path)
+test_configuration = load_configuration_template(configs_path, config_parameters, test_metadata)
+
+
+# Tests
+@pytest.mark.parametrize('test_configuration, test_metadata', zip(test_configuration, test_metadata), ids=test_cases_ids)
+def test_invalid(test_configuration, test_metadata, daemons_handler, set_wazuh_configuration, configure_local_internal_options,
+                 truncate_monitored_files):
+    '''
+    description: Check if the 'github' module detects invalid configurations. For this purpose, the test
+                 will configure that module using invalid configuration settings with different attributes.
+                 Finally, it will verify that error events are generated indicating the source of the errors.
+
+    wazuh_min_version: 4.3.0
+
+    tier: 0
+
+    parameters:
+        - test_configuration:
+            type: data
+            brief: Configuration used in the test.
+        - test_metadata:
+            type: data
+            brief: Configuration cases.
+        - daemons_handler:
+            type: fixture
+            brief: Manages daemons to reset Wazuh.
+        - set_wazuh_configuration:
+            type: fixture
+            brief: Configure a custom environment for testing.
+        - configure_local_internal_options:
+            type: fixture
+            brief: Set internal configuration for testing.
+        - truncate_monitored_files:
+            type: fixture
+            brief: Reset the 'ossec.log' file and start a new monitor.
+
+    assertions:
+        - Verify that the 'github' module generates error events when invalid configurations are used.
+
+    input_description: A configuration template (github_integration) is contained in an external YAML file
+                       (wazuh_conf.yaml). That template is combined with different test cases defined in
+                       the module. Those include configuration settings for the 'github' module.
+
+    expected_output:
+        - r'wm_github_read(): ERROR.* Invalid content for tag .*'
+        - r'wm_github_read(): ERROR.* Empty content for tag .*'
+
+    tags:
+        - invalid_settings
+    '''
+    wazuh_log_monitor = FileMonitor(WAZUH_LOG_PATH)
+    
+    # Configuration error -> ValueError raised
+    try:
+        control_service('restart')
+    except ValueError:
+        pass
+
+    detect_wrong_content_config(test_metadata['error_type'], test_metadata['event_monitor'], 'github', wazuh_log_monitor)
```

Comment:
Check: https://github.com/wazuh/wazuh/blob/4157_tests_migration/tests/integration/test_analysisd/test_signature_id/test_invalid_signature_id.py#L124

Reply:
Fixed with this [commit](https://github.com/wazuh/wazuh/pull/17621/commits/d838224bd5ffa7fad4853f1a299264bb9371c533)